sourcegraph/sourcegraph
# notebooks: symbol block frontend

TODO:

- [x] Serializing to Markdown
- [x] Link to symbol in the block

https://user-images.githubusercontent.com/6417322/157464054-43ea924b-80b4-4a5b-8296-82470b322df9.mov

## Test plan



* Create a new notebook and add a symbol block
* Find a symbol you would like to track using a Sourcegraph query
* Click on the symbol you would like to track
* You should see the symbol information displayed in the header
* You should see the highlighted code appear and you should see the symbol highlighted in the code
* Code intel should work in the highlighted code snippet
